File: commons/src/main/java/com/sdex/commons/BaseDialogFragment.java
package com.sdex.commons;

import androidx.fragment.app.DialogFragment;
import androidx.fragment.app.FragmentManager;
import android.util.Log;

public class BaseDialogFragment extends DialogFragment {

  private static final String TAG = "BaseDialogFragment";

  @Override
  public void show(FragmentManager manager, String tag) {
    try {
      super.show(manager, tag);
    } catch (IllegalStateException e) {
      Log.d(TAG, "Failed to add fragment", e);
    }
  }
}

File: app/src/main/java/com/sdex/activityrunner/commons/BaseDialogFragment.kt
package com.sdex.activityrunner.commons

import androidx.fragment.app.DialogFragment
import androidx.fragment.app.FragmentManager

open class BaseDialogFragment : DialogFragment() {

    override fun show(manager: FragmentManager, tag: String?) {
        try {
            super.show(manager, tag)
        } catch (_: IllegalStateException) {
        }
    }
}
